seed/challenges/01-front-end-development-certification/basic-javascript.json

```diff
@@ -3458,6 +3456,9 @@
         "assert(code.match(/break/g).length > 2, 'message: You should have at least 3 <code>break</code> statements');"
       ],
       "type": "waypoint",
+      "MDNlinks": [
```

Comment:
Well in that case if you can rebase then its good. :confused: 
Or I would recommend making a fresh branch of the latest staging and adding your test case.

Just make sure you save your fix somewhere before hand.
I am sure you have gone through this but just in case : [How to maintain a local instance](https://github.com/FreeCodeCamp/FreeCodeCamp/wiki/How-To-Fork-And-Maintain-a-Local-Instance-of-Free-Code-Camp)

Not sure if this was because of COM1000, dun have much idea on how it works.